javascript/javascript3/week2/main.js

```diff
@@ -0,0 +1,22 @@
+async function translateOneByOne() {
+    try {
+        await moveElement(redBox, { x: 20, y: 300 });
+        console.log("Red circle moved");
+        await moveElement(blueBox, { x: 400, y: 300 });
+        console.log("Blue circle moved");
+        await moveElement(greenBox, { x: 400, y: 20 });
+        console.log("green circle moved");
+    } catch (error) {
+        console.log(error);
+    }
+}
+translateOneByOne();
+
+async function translateAllAtOnce() {
```

Comment:
I have run the application and the balls don't seem to move to the expected circles.

Reply:
I think it will run now. Thankyou.